### ISSUE TITLE
Sensible World of Soccer '96-'97 not working

### ISSUE THREAD

**Issue by ifighftdragons** [Mazamars312/Analogue-Amiga#5]
Sensible World of Soccer '96-'97 gives this error:

type failed returncode 20

**Comment by hallucino** [comment]
It is working for me, it is a little bit slow to load but works great

**Comment by ifighftdragons** [comment]
> It is working for me, it is a little bit slow to load but works great

Would you mind sharing exactly how you got it to work, like various settings and disk drive setup? 

And are you sure you're specifically playing SWOS 96-97, and not one of the many other SWOS games?

**Comment by hallucino** [comment]
I have Sensible World of Soccer (1996)(-)(Disk 1 of 2) and Disk 2/2.

You need to change the RAM to 1MB and load the first disk (disk 0) then it will ask you to load the second disk (disk 0), that's all I did.

**Comment by ifighftdragons** [comment]
> Sensible World of Soccer (1996)(-)(Disk 1 of 2)

So, that's not the same game. You're playing the regular Sensible World of Soccer game, re-released in '96.

The game I'm talking about is specifically Sensible World of Soccer '96-'97. (Because it's from the '96-'97 soccer season).
This is the latest and best version of SWOS.

Can you try _that_ game, and see if you can get it to work?

Thanks for trying to help.

**Comment by hallucino** [comment]
Any hints to get a rom that works in classic emulator ?

**Comment by ifighftdragons** [comment]
Not sure how that is relevant to getting it to work on Analogue Pocket? Plus, it's not allowed to discuss rom sites and downloads.

However, you can easily find it by using google and keywords such as the correct game title + adf rom

**Comment by hallucino** [comment]
I'm getting an error too with this version

**Comment by terminator2k2** [comment]
have you tried this maz updated the core? seems to work fine here...
![20221213_193003](https://user-images.githubusercontent.com/48845336/207429029-0616c785-4e34-4492-8e60-3dfdd6b9bfed.png)
![20221213_193323](https://user-images.githubusercontent.com/48845336/207429032-de8c9b48-aba8-46a5-a8ff-65e64b39eaef.png)

**Comment by ifighftdragons** [comment]
> have you tried this since maz updated the core? seems to work fine here... ![20221213_193003](https://user-images.githubusercontent.com/48845336/207429029-0616c785-4e34-4492-8e60-3dfdd6b9bfed.png) ![20221213_193323](https://user-images.githubusercontent.com/48845336/207429032-de8c9b48-aba8-46a5-a8ff-65e64b39eaef.png)

That's awesome, if it actually works now.

What version of the kickstart are you using?
And what are your other settings? Would you mind listing them so I can replicate?

**Comment by ifighftdragons** [comment]
> name of disk: Sensible World of Soccer '96-'97 (1996-11-06)(Renegade)(Disk 1 of 2)[cr HF][f Codetapper].adf
> 
> kickstart 1.3 OG/Turbo/Pal 68k cpu 2mb chip 1.5mb slow 8mb/none fast
> 
> this one gave me an error on kick 1.3 :
> 
> Sensible World of Soccer '96-'97 (1996-11-06)(Renegade)(Disk 1 of 2)[cr HF][KS2.0].adf
> 
> but works ok with kickstart 3.1.4 ( take it [KS2.0] means kickstart 2.0 or higher)
> 
> this one gives guru med red screen of death
> 
> Sensible World of Soccer '96-'97 (1996-11-06)(Renegade)(Disk 1 of 2)[cr HF][update data disk][req. old swos disk 1].adf

I'll number my feedback, to make the long messages more easy to digest:
1. What's your settings for "Installed Floppy Disks" ?
2. Do you load this as "Load Floppy Drive 0" ?
3. Since this is a multi-disk game: What happens when it's time to load Disk 2/2 ? Or can you just play the entire game with the first disk?
4. You mentioned 2 specific versions that worked? I'm not bad at finding games or googling, but I honestly can't find either of these:
_Sensible World of Soccer '96-'97 (1996-11-06)(Renegade)(Disk 1 of 2)[cr HF][f Codetapper].adf
Sensible World of Soccer '96-'97 (1996-11-06)(Renegade)(Disk 1 of 2)[cr HF][KS2.0].adf_
I can find similarly named versions, but none with Codetapper og KS2.0 in the title.
I can find a couple of mentions of them in old forums, but no place to actually find them.
I have very similarly titled .adf files, but not identical. They will not run.

**Comment by terminator2k2** [comment]
are you on classic gaming  or fpgaming discord at all ?

**Comment by ifighftdragons** [comment]
I'm not on discord

**Comment by terminator2k2** [comment]
ah ok, could of pm'd you

**Comment by ifighftdragons** [comment]
Right on.

What's your settings for "Installed Floppy Disks" ? It's where you can select number of drives and their speed.

If I could just find a workable .adf now, I'd be off and running.

**Comment by terminator2k2** [comment]
4 drive 2x speed

**Comment by ifighftdragons** [comment]
I set up a temporary chat room so we can resolve this quickly =)

https://stin.to/ntyub (perfectly safe, btw)

**Comment by Mazamars312** [comment]
This was one thing I hated about the amiga, there are so many configs that are required for each game including bios

**Comment by ifighftdragons** [comment]
> This was one thing I hated about the amiga, there are so many configs that are required for each game including bios

Absolutely. Can't believe all the hoops I've jumped through to get some games running. And not without help from you and terminator2k2!

Hopefully the core will further evolve into an easier Amiga life, as has already proven to be the case. Great job! Sorry for off-topic rant.

**Comment by terminator2k2** [comment]
> What are the second adf you use with "Sensible World of Soccer '96-'97 (1996-11-06)(Renegade)(Disk 1 of 2)[cr HF][f Codetapper].adf" ?

Sensible World of Soccer '96-'97 (1996-11-06)(Renegade)(Disk 2 of 2)[cr HF].adf

**Comment by limi** [comment]
For what it’s worth, this game seems to work perfectly using the https://amiga.vision setup if you don’t want to deal with all the configuration yourself.

**Comment by Romanista345** [comment]
@limi unfortunately it doesn't save and load doesn't work, while it worked it the previous setup, I filed a ticket..
